bundles/org.openhab.binding.solarforecast/src/main/resources/OH-INF/config/fs-site-config.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:solarforecast:fs-site">
+		<parameter name="location" type="text">
+			<context>location</context>
+			<label>PV Location</label>
+			<description>Location of photovoltaic system</description>
```

Comment:
Described in readme
```
### ForecastSolar Bridge Configuration

| Name                   | Type    | Description                           | Default      | Required |
|------------------------|---------|---------------------------------------|--------------|----------|
| location               | text    | Location of Photovoltaic system.      | empty        | no       |
| apiKey                 | text    | API Key                               | N/A          | no       |

`location` defines latitude, longitude values of your PV system.
In case of empty the location configured in openHAB is obtained.
```
Added also in description